tests/integration_tests/test_chat_completion.py

```diff
@@ -202,11 +208,21 @@ def is_ai21(deployment: ChatCompletionDeployment) -> bool:
 
 
 def is_vision_model(deployment: ChatCompletionDeployment) -> bool:
-    return is_claude3(deployment) or deployment in [
+    allowed_models = [
         ChatCompletionDeployment.META_LLAMA3_2_11B_INSTRUCT_V1,
         ChatCompletionDeployment.META_LLAMA3_2_90B_INSTRUCT_V1,
     ]
 
+    excluded_models = {
```

Comment:
Leave a comment that `Claude 3.5 Haiku was launched as a text-only model` and leave a reference to the [model card](https://assets.anthropic.com/m/61e7d27f8c8f5919/original/Claude-3-Model-Card.pdf)